cockatrice/src/chatview.cpp

```diff
@@ -155,12 +156,14 @@ void ChatView::appendMessage(QString message, RoomMessageTypeFlags messageType,
         senderFormat.setAnchorHref("user://" + QString::number(userLevel) + "_" + sender);
         if (sameSender) {
             cursor.insertText("    ");
-        } else {
+        }
+        else {
             if (!sender.isEmpty() && tabSupervisor->getUserListsTab()) {
                 const int pixelSize = QFontInfo(cursor.charFormat().font()).pixelSize();
                 QMap<QString, UserListTWI *> buddyList = tabSupervisor->getUserListsTab()->getBuddyList()->getUsers();
-                cursor.insertImage(UserLevelPixmapGenerator::generatePixmap(pixelSize, userLevel,
-                                                                            buddyList.contains(sender)).toImage());
+                QMap<QString, UserListTWI *> userList = tabSupervisor->getUserListsTab()->getAllUsersList()->getUsers();
```

Comment:
I'm fine with this as is until we can come up with a better solution in the future. No need to withhold this PR for it.